deps/v8/include/v8-inspector.h

```diff
@@ -105,8 +105,9 @@ class V8_EXPORT V8StackTrace {
   virtual StringView topSourceURL() const = 0;
   virtual int topLineNumber() const = 0;
   virtual int topColumnNumber() const = 0;
-  virtual StringView topScriptId() const = 0;
-  virtual int topScriptIdAsInteger() const = 0;
+  virtual int topScriptId() const = 0;
+  V8_DEPRECATE_SOON("Use V8::StackTrace::topScriptId() instead.")
+  int topScriptIdAsInteger() const { return topScriptId(); }
```

Comment:
This file is not exposed, right?

Reply:
[It's not exposed](https://github.com/nodejs/node/blob/b6d4e2797a761794e292a664ee92823034c17c88/tools/install.py#L161-L172)